src/js/tracing/reactnativetracing.ts

```diff
@@ -255,6 +262,9 @@ export class ReactNativeTracing implements Integration {
       logger.log('[ReactNativeTracing] Not instrumenting route changes as routingInstrumentation has not been set.');
     }
 
+    this._getCurrentViewEventProcessor = this._getCurrentViewEventProcessor.bind(this);
+    addGlobalEventProcessor(async (event: Event) => this._getCurrentViewEventProcessor(event));
```

Comment:
We can right away pass the function with binded this as a global event processor. And avoid creating the arrow function. 

```suggestion
    addGlobalEventProcessor(this._getCurrentViewEventProcessor.bind(this));
```